### PR TITLE
avoid releasing builtin asset

### PR DESCRIPTION
**Pull request by SantyWang**
Re: cocos-creator/2d-tasks#
https://forum.cocos.com/t/ccc2-1-2/80789
Changes:
 * 避免释放内部资源

### CODE REVIEW

File: cocos2d/core/platform/CCAssetLibrary.js [hunk 1]
```diff
@@ -399,6 +399,8 @@ let _builtins = {
     material: {}
 };
 
+let builtinDeps = {};
```
Comment (2youyou2) [inline cocos2d/core/platform/CCAssetLibrary.js]: 是否应该跟 _builtins 一样用下划线标记

File: cocos2d/core/platform/CCAssetLibrary.js [hunk 2]
```diff
@@ -445,5 +450,8 @@ AssetLibrary.resetBuiltins = function () {
         material: {}
     };
 };
+AssetLibrary.getBuiltinDeps = function () {
+    return builtinDeps;
```
Comment (2youyou2) [inline cocos2d/core/platform/CCAssetLibrary.js]: resetBuiltins 里面应该也要重置 builtinDeps

Comment (SantyWang) [reply cocos2d/core/platform/CCAssetLibrary.js]: 有道理

File: cocos2d/core/platform/CCAssetLibrary.js (hunk 1 above)
Comment (SantyWang) [reply cocos2d/core/platform/CCAssetLibrary.js]: 👌

**Pull request merged**